### PR TITLE
Bug 1205457 - Ukrainian localization

### PR DESCRIPTION
**Pull request by stenox**

### CODE REVIEW

**Comment by dylano** [comment]
Thanks for the PR! I'm hoping to publish an update soon to pull in some more languages and would like to include this. Can you also add a [uk] section to /manifest.webapp?

**Comment by stenox** [comment]
Ukrainian section in /manifest.webapp is done.
Please check.
Thanks!

File: locales/locales.ini [hunk 1]
```diff
@@ -20,3 +20,6 @@
 
 [nl]
 @import url(notes.nl.properties)
+
+[uk]
+@import url (notes.uk.properties)
```
Comment (dylano) [inline locales/locales.ini]: error -- space after "url" prevents loading of the property file

Comment (dylano) [inline locales/locales.ini]: btw, with Firefox WebIDE you can install and test this app on the FxOS simulator to ensure that strings are loading and displaying properly

Comment (stenox) [inline locales/locales.ini]: Oops, already fixed.
Ok, I'll test it and inform about the result here.
Thanks!

**Comment by dylano** [comment]
For reference: https://bugzilla.mozilla.org/show_bug.cgi?id=1205457

Can you update the title on this pr to include the bug number, e.g. "Bug 1205457 - Ukrainian localization"

Thanks!

**Comment by stenox** [comment]
Packaged app successfully tested on my Flame device.
All translations looks well.

**Pull request merged**